frontend/packages/kubevirt-plugin/src/components/vms/vm.tsx

```diff
@@ -167,18 +189,22 @@ type VMRowProps = {
   style: object;
   customData: {
     pods: PodKind[];
-    migrations: any[];
+    migrations: K8sResourceKind[];
+    migrationLookup: K8sEntityMap<VMIKind>;
+    vmiLookup: K8sEntityMap<VMIKind>;
   };
 };
 
 type VMListProps = {
   data: VMKind[];
   resources: {
     pods: FirehoseResult<PodKind[]>;
-    migrations: FirehoseResult<any[]>;
+    migrations: FirehoseResult<K8sResourceKind[]>;
```

Comment:
FIY `K8sResourceKind[]` is default so it can be shorter
```suggestion
    migrations: FirehoseResult;
```

Reply:
this is done to be more explicit + so the types can be easily replaced with correct ones in the future